doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'Ansible Guide'
-copyright = '2017, Adfinis SyGroup AG'
+copyright = '2019, Adfinis SyGroup AG'
```

Comment:
```suggestion
copyright = '2017 - 2019, Adfinis SyGroup AG'
```
?

Or (my personal fave):
```suggestion
copyright = 'Adfinis SyGroup AG'
```

Reply:
Good point, I don't like to maintain such stuff neither. :)